.gitignore

```diff
@@ -0,0 +1,65 @@
+# .gitignore file
+
+# Backup files
+*~
+
```

Comment:
No `__pycache__`?

Reply:
I just copied it from pydicom and added a little bit extra

Reply:
It would be good to add it then.

Reply:
Done